Dockerfile-python

```diff
@@ -1,5 +1,24 @@
-FROM python:3
+FROM python:3.5
+RUN pip install togeojsontiles
 RUN pip install postgres
 RUN pip install psycopg2
 RUN apt update
-RUN apt install -y postgresql 
\ No newline at end of file
+RUN apt install -y postgresql 
+
+RUN apt-get update \
+  && apt-get -y upgrade \
+  && apt-get -y install git build-essential libsqlite3-dev zlib1g-dev
+
+# Create a directory and copy in all files
+RUN mkdir -p /tmp/tippecanoe-src
+RUN git clone https://github.com/mapbox/tippecanoe.git /tmp/tippecanoe-src
+WORKDIR /tmp/tippecanoe-src
```

Comment:
Do check if we can install tippecanoe directly instead of cloning and then building it inside the container.

Reply:
This is the best way to install tippecanoe